app/javascript/utilities/preact/index.js

```diff
@@ -0,0 +1,20 @@
+import { render as preactRender } from 'preact';
+import { unmountComponentAtNode } from 'preact/compat';
+
+/**
+ * Renders a Preact component into the given parent DOM element.
+ *
+ * Note: This function is calling Preact's render under the hood along with
+ * a call to InstantClick's change event that explicitly unmounts the component rendered.
+ *
+ * @param {ComponentChild} vnode
+ * @param {Element | Document | ShadowRoot | DocumentFragment} parent
+ * @param {Element | Text} replaceNode
+ */
+export function render(vnode, parent, replaceNode) {
```

Comment:
This is the magic sauce.

Reply:
> My only reservation about having this function is it hides what's really going on.

💯 Not a big fan tbh, but I also don't have a better suggestion. Except maybe for making the intention of this a bit more explicit where it's used, at the cost of slightly longer imports:

```js
import { instantClickRender as render } from '@utilities/preact';
```

Reply:
I think explicitly calling it `instantClickRender` makes sense. I'll think of some other possible ways to approach it his as well because of my reservations and your "💯 not a fan" 😎

Reply:
I went with the rename to `instantClickRender`.